src/dmd/e2ir.d

```diff
@@ -6228,6 +6228,7 @@ elem *toElemStructLit(StructLiteralExp sle, IRState *irs, TOK op, Symbol *sym, b
 
     // CTFE may fill the hidden pointer by NullExp.
     {
+        VarDeclaration vbf;
```

Comment:
Shouldn't this be near line 6274? It should increase readability

Reply:
It needs to be outside the loop.